app/src/main/java/com/example/playlist/ui/mediaActivity/viewModel/FavoriteFragmentViewModel.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.playlist.ui.mediaActivity.viewModel
+
+import androidx.lifecycle.ViewModel
+import org.koin.core.component.KoinComponent
+
+class FavoriteFragmentViewModel : ViewModel(), KoinComponent {
```

Comment:
```suggestion
class FavoriteFragmentViewModel : ViewModel(){
```
Это лишнее, вьюмодель не провайдит никакие зависимости, она сама является зависимостью